### PR TITLE
Added services to return count

### PR DESCRIPTION
**Pull request by sumantsahney**
Added services to return count

### CODE REVIEW

**Comment by sumantsahney** [comment]
Can you please review it ?

**Review by Code4SocialGood — COMMENTED** [review]
1. Could you please remove file: c4sg-services.iml
2. Do we need to use native query for count?
3. The method signature for controller: do we need to use map? can we simply return the count?

File: c4sg-services.iml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (felipealvesgnu) [inline c4sg-services.iml:1]: Don't commit this file, delete it

File: src/main/java/org/c4sg/controller/OrganizationController.java [hunk 2]
```diff
@@ -248,4 +248,15 @@ public void approve(
 
 		return new ResponseEntity<>(map, HttpStatus.OK);
 	}
+
+	@CrossOrigin
+	@RequestMapping(value="/active/total", method = RequestMethod.GET)
+	@ApiOperation(value = "Total number of active organization")
+	public ResponseEntity<Map<String, String>> getTotalActiveOrg() {
+		HashMap<String, String> map = new HashMap<>();
```
Comment (felipealvesgnu) [inline src/main/java/org/c4sg/controller/OrganizationController.java:256]: As a rule of polymorphism.
Use `Map<String, String> map = new HashMap<>();`

File: src/main/java/org/c4sg/controller/ProjectController.java [hunk 3]
```diff
@@ -386,5 +387,16 @@ public void saveImage(
     	
         return projectService.findJobTitles();
     }
+
+    @CrossOrigin
+    @RequestMapping(value="/active/total", method = RequestMethod.GET)
+    @ApiOperation(value = "Total number of active projects")
+    public ResponseEntity<Map<String, String>> getTotalActiveProjects() {
+        HashMap<String, String> map = new HashMap<>();
```
Comment (felipealvesgnu) [inline src/main/java/org/c4sg/controller/ProjectController.java:395]: Use `Map<String, String> map = new HashMap<>();`

File: src/main/java/org/c4sg/controller/UserController.java [hunk 4]
```diff
@@ -188,4 +192,14 @@ public void saveAvatar(
 
     }
 
+    @CrossOrigin
+    @RequestMapping(value="/totalVolunteers", method = RequestMethod.GET)
+    @ApiOperation(value = "Total number of Volunteers")
+    public ResponseEntity<Map<String, String>> getTotalVolunteers() {
+        HashMap<String, String> map = new HashMap<>();
```
Comment (felipealvesgnu) [inline src/main/java/org/c4sg/controller/UserController.java:199]: Use` Map<String, String> map = new HashMap<>();`

**Review by felipealvesgnu — CHANGES_REQUESTED** [review]

**Pull request closed without merging**